packages/app-desktop/integration-tests/pluginApi.spec.ts

```diff
@@ -30,7 +30,7 @@ test.describe('pluginApi', () => {
 		await mainScreen.createNewNote('First note');
 
 		const editor = mainScreen.noteEditor;
-		await editor.expectToHaveText('');
+		await editor.expectToHaveText('\n');
```

Comment:
This change in expected test output is required due to a change to `editor.expectToHaveText`. Previously, newlines were not included in the content fetched from the editor.